src/main/java/org/opensearch/ubi/UserBehaviorInsightsPlugin.java

```diff
@@ -35,10 +35,7 @@
 import org.opensearch.ubi.model.SettingsConstants;
 import org.opensearch.watcher.ResourceWatcherService;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
```

Comment:
Is * a normal pattern in OPenSearch?   (would not be in Solr!)

Reply:
Not sure, but I don't like it either. Will update.

Reply:
Turned out to be several instances of that so I fixed them all. There's a setting in IntelliJ to not let it do that.